test/cadet/assessments_test.exs

```diff
@@ -0,0 +1,200 @@
+defmodule Cadet.AssessmentsTest do
+  use Cadet.DataCase
+
+  alias Cadet.Assessments
+  alias Cadet.Accounts
+
+  test "all missions" do
+    missions = [
+      insert(:mission),
+      insert(:mission),
+      insert(:mission),
+      insert(:mission),
+      insert(:mission)
+    ]
+
+    result = Assessments.all_missions()
+    assert Enum.all?(result, fn x -> x.id in Enum.map(missions, fn m -> m.id end) end)
+  end
+
+  test "all open missions" do
+    open_mission = insert(:mission, is_published: true, category: :mission)
+    closed_mission = insert(:mission, is_published: false, category: :mission)
+    result = Enum.map(Assessments.all_open_missions(:mission), fn m -> m.id end)
+    assert open_mission.id in result
+    refute closed_mission.id in result
+  end
+
+  test "create mission" do
+    {:ok, mission} =
+      Assessments.create_mission(%{
+        title: "mission",
+        category: :mission,
+        open_at: Timex.now(),
+        close_at: Timex.shift(Timex.now(), days: 7)
+      })
+
+    assert mission.title == "mission"
```

Comment:
Change all the `assert`s done in this style to pattern matching instead, e.g. `assert mission = %{title: "mission", category: :mission}`